packages/@ourworldindata/grapher/src/stackedCharts/StackedBarChart.tsx

```diff
@@ -152,6 +162,14 @@ export class StackedBarChart
         return GRAPHER_FONT_SCALE_12 * this.baseFontSize
     }
 
+    @computed protected get yAxisDomain(): [number, number] {
+        const yValues = this.allStackedPoints.map(
+            (point) => point.value + point.valueOffset
+        )
+        const hasNegativeValues = yValues.some((value) => value < 0)
+        return [hasNegativeValues ? min(yValues) ?? 0 : 0, max(yValues) ?? 0]
```

Comment:
This'll go wrong if all values are negative, I think (because then `max(yValues) < 0`). And in theory, you don't need the extra `hasNegativeValues` check either:

```suggestion
        return [min([0, ...yValues]) ?? 0, max([0, ...yValues]) ?? 0]
```

I think this should work 🤞🏻